nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/api/WorkManager.java

```diff
@@ -138,6 +138,12 @@
      */
     boolean isProcessingEnabled();
 
+    /**
+     * @return true if the implementation supports processing disabling
+     * @since 10.3
+     */
+    boolean supportsProcessingDisabling();
```

Comment:
This method is just use in the test to do an assume. Could it be removed there and replaced by an `instanceof` in the test ? (in order to not add a new API to this heavy interface).

Reply:
Not possible due to WorkManagerImpl which could support or not this feature.